app/assets/stylesheets/header.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+  height: 30px;
+  background-color: rgba(0,0,0,0.75);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.75)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by a single space